### ISSUE TITLE
ligand always out of box in ligand_in_water.ipynb

### ISSUE THREAD

**Issue by ivandon15** [openforcefield/openff-interchange#924]
**Description**
Hello, developers of OpenFF,

When I was using _examples/ligand_in_water/ligand_in_water.ipynb_, I noticed that in the simulation results saved in the pdb file, the solute always remained on the outside edge of the water box (img shown below). Is this correct? 
<img width="289" alt="image" src="https://github.com/openforcefield/openff-interchange/assets/45418272/37c7d830-5084-42ec-84b2-4e6356c6fc7b">

Because when I use traditional MD simulations, the solute is usually inside the solvent box (occasionally at the box edge). I have checked all current issues and haven't found an answer to my confusion. Have I missed something? 

Thank you very much.

**Comment by mrshirts** [comment]
I'm **guessing** this has to do with periodic boundary conditions; the ligand is being draw with the first atom at 0,0,0, but happens to be built with negative coordinates, whereas the water is filled in with positive coordinates.   You would probably need to post the actually pdb to see.

**Comment by mattwthompson** [comment]
Yes this looks like a quirk of visualization. Visualization/analysis tools usually have ways to move molecules between periodic images.

If the simulation runs, it's not likely an issue of the system being prepared wrong.

**Comment by ivandon15** [comment]
Thanks for your response! 

I have uploaded the pdb file for reference at this [repo link](https://github.com/ivandon15/syn_data/blob/main/trajectory.pdb), where I only changed the _step_ parameter in the _ipynb_ file (from 5000 to 10000, hence there will be 20 MODELS in the pdb file). 

From the pdb, it can be seen that the ligand is outside the edge of the water box in all 20 MODELS. However, as you mentioned, the simulation did not encounter any problems, so I wonder if I can disregard this odd visualization issue.
